client/signup/steps/woocommerce-install/step-business-info/index.tsx

```diff
@@ -0,0 +1,226 @@
+						] }
+						onChange={ updateProductCount }
+					/>
+
+					<SelectControl
+						label={ __( 'Currently selling elsewhere? (optional)', 'woocommerce-admin' ) }
+						value={ getProfileValue( 'selling_venues' ) }
+						options={ [
+							{ value: '', label: '' },
+							{ value: 'no', label: __( 'No', 'woocommerce-admin' ) },
+							{ value: 'other', label: __( 'Yes, on another platform', 'woocommerce-admin' ) },
+							{
+								value: 'other-woocommerce',
+								label: __(
+									'Yes, I own a different store powered by WooCommerce',
+									'woocommerce-admin'
+								),
+							},
+							{
+								value: 'brick-mortar',
+								label: __( 'Yes, in person at physical stores and/or events', 'woocommerce-admin' ),
+							},
+							{
+								value: 'brick-mortar-other',
+								label: __(
+									'Yes, on another platform and in person at physical stores and/or events',
+									'woocommerce-admin'
+								),
+							},
+						] }
+						onChange={ updateSellingVenues }
+					/>
+
+					{ [ 'other', 'brick-mortar-other' ].includes( getProfileValue( 'selling_venues' ) ) && (
+						<>
+							<SelectControl
+								label={ __( 'Which platform is the store using? (optional)', 'woocommerce-admin' ) }
+								value={ getProfileValue( 'other_platform' ) }
+								options={ [
+									{ value: '', label: '' },
+									{
+										value: 'shopify',
+										label: __( 'Shopify', 'woocommerce-admin' ),
+									},
+									{
+										value: 'bigcommerce',
+										label: __( 'BigCommerce', 'woocommerce-admin' ),
+									},
+									{
+										value: 'magento',
+										label: __( 'Magento', 'woocommerce-admin' ),
+									},
+									{
+										value: 'wix',
+										label: __( 'Wix', 'woocommerce-admin' ),
+									},
+									{
+										value: 'amazon',
+										label: __( 'Amazon', 'woocommerce-admin' ),
+									},
+									{
+										value: 'ebay',
+										label: __( 'eBay', 'woocommerce-admin' ),
+									},
+									{
+										value: 'etsy',
+										label: __( 'Etsy', 'woocommerce-admin' ),
+									},
+									{
+										value: 'squarespace',
+										label: __( 'Squarespace', 'woocommerce-admin' ),
+									},
+									{
+										value: 'other',
+										label: __( 'Other', 'woocommerce-admin' ),
+									},
+								] }
+								onChange={ updateOtherPlatform }
+								required
+							/>
+
+							{ getProfileValue( 'other_platform' ) === 'other' && (
+								<TextControl
+									label={ __( 'What is the platform name? (optional)', 'woocommerce-admin' ) }
+									onChange={ updateOtherPlatformName }
+									value={ getProfileValue( 'other_platform_name' ) }
+								/>
+							) }
+						</>
+					) }
+
+					<ActionSection>
+						<SupportCard />
+						<StyledNextButton
+							onClick={ () => {
+								dispatch( submitSignupStep( { stepName: 'business-info' } ) );
+								updateOnboardingProfile( 'completed', true );
```

Comment:
This doesn't seem to hide the setup-wizard but I think should?

If we can get this to work is the business step the right place for this or should it happen in the transfer step?

Reply:
Yes, this could be due to waiting on the Atomic Jetpack release.

> If we can get this to work is the business step the right place for this or should it happen in the transfer step?

🤔 I feel like saving it with the business profile data makes sense, if we consider it a marker that all the profile data is collected, not necessarily that the transfer or setup completed.

> This may also mean we can drop Automattic/wc-calypso-bridge#734 completely which would be a nice result.

Oh, that would be great!

Reply:
> I feel like saving it with the business profile data makes sense

That was my original thought and then we added the email field in the address step :) 
I don't feel super strongly about it. The transfer step probably makes sense with the flow being complete then.